learningresources/models.py

```diff
@@ -216,12 +216,25 @@ def get_preview_url(resource, org=None, course_number=None, run=None):
         run=run,
     )
 
-    if resource.url_name is None:
+    url_name = None
```

Comment:
nit:

Lines 219 - 221 can just be `url_name = resource.url_name`, and line 222 can be `if url_name is None`. The `else` makes it _slightly_ more difficult to follow.